### PR TITLE
fix: enhance design of video-panel button

### LINKED ISSUE #6231: Video Panel Button: Further Enhance Design
Please change the button on the side panel in a way so it looks more in sync with the platform as follows:
- position it so it looks like in the middle of the top bar.
- Change button to: width: 32px, height: 50px
- move the "eventyay" text to the right, so buttons do not overlap the text (also increase the area to the right side border, so it looks like positioned in the middle of the area even if the button is not there)
- move the event name on the side panel itself a slight bit up
- on other pages these kind of buttons sometimes pop out a bit as an effect. Please implement such an effect every 30 seconds

Screenshot from 2021-01-04 13-58-23

Screenshot from 2021-01-04 13-58-41

### PR DESCRIPTION
**Pull request by ranjitkshah**
Fixes #6231 

#### Short description of what this resolves:
Screenshot:
image

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by maze-runnar** [comment]
> move the event name on the side panel itself a slight bit up

Screenshot from 2021-01-05 10-15-18

File: app/styles/components/stream.scss [hunk 1]
```diff
@@ -26,9 +26,10 @@
 
 .stream-side-panel-button {
   position: fixed;
-  top: 1rem;
+  top: 3px;
```
Comment (Creatoon) [inline app/styles/components/stream.scss:29]: you should write this is in rem in order to make it relative

**Comment by ranjitkshah** [comment]
screenshot:
image

**Review by mariobehling — CHANGES_REQUESTED** [review]
Thanks, please also change the following:
* center the < sign in the center of the icon when the panel is opened 
Screenshot from 2021-01-10 05-42-11
* Do not move up the event name heading as much. Leave more space above the event name (compare to sample screenshot in the issue itself)

**Comment by ranjitkshah** [comment]
@mariobehling pls check now thanks

image

**Review by mariobehling — APPROVED** [review]
Looks good to me. Thanks.

**Pull request merged**